scripts/ts-node.js

```diff
@@ -9,7 +9,13 @@
 //
 //      yarn ts-node
 
+require('dotenv').config()
 require('ts-node').register({
-  compilerOptions: { module: 'commonjs', esModuleInterop: true },
+  compilerOptions: {
+    module: 'commonjs',
+    esModuleInterop: true,
+    noUnusedLocals: false,
+    strict: false,
```

Comment:
Why are these required? I'd think it should pick up the correct tsconfig of the executing file.